src/coreclr/clrdefinitions.cmake

```diff
@@ -104,9 +104,7 @@ add_definitions(-DFEATURE_DEFAULT_INTERFACES)
 if(FEATURE_EVENT_TRACE)
     add_compile_definitions($<$<NOT:$<BOOL:$<TARGET_PROPERTY:CROSSGEN_COMPONENT>>>:FEATURE_EVENT_TRACE>)
     add_definitions(-DFEATURE_PERFTRACING)
-    if(FEATURE_PERFTRACING_C_LIB)
-      add_definitions(-DFEATURE_PERFTRACING_C_LIB)
-    endif(FEATURE_PERFTRACING_C_LIB)
+    add_definitions(-DFEATURE_PERFTRACING_C_LIB)
```

Comment:
If `FEATURE_PERFTRACING` and `FEATURE_PERFTRACING_C_LIB` are equivalent with this change, maybe we can delete the latter?